src/platform/imx8m/lib/dai.c

```diff
@@ -18,6 +18,17 @@ static SHARED_DATA struct dai sai[] = {
 	.index = 3,
 	.plat_data = {
 		.base = SAI_3_BASE,
```

Comment:
This can go as a separate PR. Please add some more info in the commit message. The current commit message is not very informative.

Reply:
Will do.